bundles/org.openhab.binding.keba/src/main/resources/OH-INF/thing/kecontact.xml

```diff
@@ -119,7 +120,7 @@
 		<item-type>Switch</item-type>
 		<label>Enabled</label>
 		<description>Activation state of the wallbox</description>
-		<state readOnly="false"></state>
+		<state readOnly="true"></state>
```

Comment:
It is not read only, at least for enableduser at commands are handled on that channel.

Reply:
Fixed